server/resourcemanager/testutil/test_resource.go

```diff
@@ -110,7 +111,6 @@ func testOperation(t *testing.T, op operationTester, rt ResourceTypeTest) {
 	for _, ct := range contentTypeConverters {
 		t.Run(ct.name, func(t *testing.T) {
 			ct := ct
-			t.Parallel()
```

Comment:
By running all the tests in parallel, our database connections skyrocket sometimes. By removing serializing this test we allowed our tests to have a chance to cleanup some connections.